### PR TITLE
🛠 #1808 Bootstrap user status work again

### PR DESCRIPTION
**Pull request by tissarni**
Users status is not  available anymore in Twake. The mission is to re-implement it to put it back in production

## Description

 - [x] Create new route to update status
 - [x] Create front component to change status
 - [x] Add features to just have an emoji status

 
## Related Issue

#1808

### CODE REVIEW

File: twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/CurrentUser/SaveNewStatus.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import InputWithIcon from 'app/components/Inputs/InputWithIcon';
+import classNames from 'classnames';
+
+import './SaveNewStatus.scss';
+
+type PropsType = {
+  level: any;
+};
+
+export default (props: PropsType): JSX.Element => {
```
Comment (stephanevieira75) [inline twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/CurrentUser/SaveNewStatus.tsx:16]: `export default ({ level }: PropsType): JSX.Element => { ...`

File: twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/CurrentUser/SaveNewStatus.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useCallback, useState, useRef } from 'react';
+import { Button } from 'antd';
+
+import Languages from 'services/languages/languages';
+import { useCurrentUser } from 'app/state/recoil/hooks/useCurrentUser';
+import MenusManager from 'app/components/Menus/MenusManager';
+import InputWithIcon from 'app/components/Inputs/InputWithIcon';
+import classNames from 'classnames';
```
Comment (stephanevieira75) [inline twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/CurrentUser/SaveNewStatus.tsx:8]: This should be line 3

File: twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/CurrentUser/SaveNewStatus.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+import './SaveNewStatus.scss';
+
+type PropsType = {
+  level: any;
```
Comment (stephanevieira75) [inline twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/CurrentUser/SaveNewStatus.tsx:13]: Can you try to find the level type please ?

File: twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/CurrentUser/SaveNewStatus.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  }, [status, updateStatus]);
+
+  useEffect(() => {
+    const listener = (event: { code: string; preventDefault: () => void }) => {
+      if (event.code === 'Enter' || event.code === 'NumpadEnter') {
+        console.log('Enter key was pressed. Run your function.');
+        save();
+      }
+    };
+    document.addEventListener('keydown', listener);
```
Comment (stephanevieira75) [inline twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/CurrentUser/SaveNewStatus.tsx:35]: I don't think you need this useEffect to add an event listener like this as you already use it directly line 48:

``` tsx
 <div
      onKeyDown={evt => {
        if (evt.key === 'Enter') save();
      }}
    >
....
```

**Review by stephanevieira75 — CHANGES_REQUESTED** [review]

**Comment by RomaricMourgues** [comment]
@tissarni is it OK to merge ?

**Review by RomaricMourgues — APPROVED** [review]

**Review by RomaricMourgues — APPROVED** [review]

**Pull request merged**